### PR TITLE
chore: change release workflow & logo src

### PR DESCRIPTION
**Pull request by Orkuncakilkaya**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,7 +31,7 @@
 This PHP package is automatically generated by the [Swagger Codegen](https://github.com/swagger-api/swagger-codegen) project:
 
 - API version: 3
-    - Package version: 0.2.5
+    - Package version: 1.0.0
```
Comment (ilfa) [inline README.md:34]: Looks strange, should be `0.3.0-develop.1`

File: composer.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
-    "name": "fingerprintjs/server-api-sdk",
-    "version": "0.2.5",
+    "name": "fingerprintjs/fingerprint-pro-server-api-php-sdk",
+    "version": "1.0.0",
```
Comment (ilfa) [inline composer.json:3]: The same, should be `0.3.0-develop.1`

File: config.json [hunk 3]
```diff
@@ -3,10 +3,10 @@
...
   "invokerPackage": "Fingerprintjs\\ServerAPI",
   "composerVendorName": "fingerprintjs",
-  "composerProjectName": "server-api-sdk",
+  "composerProjectName": "fingerprint-pro-server-api-php-sdk",
   "packageUrl": "https://github.com/fingerprintjs/fingerprint-pro-server-api-php-sdk",
   "gitUserId": "fingerprintjs",
   "gitRepoId": "fingerprint-pro-server-api-php-sdk",
   "description": "Fingerprint Pro Server API provides a way for validating visitors’ data issued by Fingerprint Pro.",
-  "artifactVersion": "0.2.5"
+  "artifactVersion": "1.0.0"
```
Comment (ilfa) [inline config.json:11]: And here

File: src/.travis.yml [hunk 4]
```diff
@@ -0,0 +1,10 @@
+language: php
```
Comment (ilfa) [inline src/.travis.yml:1]: Are we use travis?

File: src/.php_cs [hunk 5]
```diff
@@ -0,0 +1,18 @@
+<?php
```
Comment (ilfa) [inline src/.php_cs:1]: What the role of this file?

File: src/Configuration.php [hunk 6]
```diff
@@ -87,7 +87,7 @@ class Configuration
      *
      * @var string
      */
-    protected $userAgent = 'Swagger-Codegen/0.2.5/php';
+    protected $userAgent = 'Swagger-Codegen/1.0.0/php';
```
Comment (ilfa) [inline src/Configuration.php:90]: Same problem with wrong version

File: src/Configuration.php [hunk 7]
```diff
@@ -396,7 +396,7 @@ public static function toDebugReport()
         $report .= '    OS: ' . php_uname() . PHP_EOL;
         $report .= '    PHP Version: ' . PHP_VERSION . PHP_EOL;
         $report .= '    OpenAPI Spec Version: 3' . PHP_EOL;
-        $report .= '    SDK Package Version: 0.2.5' . PHP_EOL;
+        $report .= '    SDK Package Version: 1.0.0' . PHP_EOL;
```
Comment (ilfa) [inline src/Configuration.php:399]: And here

File: src/git_push.sh [hunk 8]
```diff
@@ -0,0 +1,60 @@
+#!/bin/sh
```
Comment (ilfa) [inline src/git_push.sh:1]: Why do we need this script?

**Review by ilfa — CHANGES_REQUESTED** [review]

File: src/.travis.yml (hunk 4 above)
Comment (Orkuncakilkaya) [reply src/.travis.yml:1]: No we are not currently using travis

File: src/.php_cs (hunk 5 above)
Comment (Orkuncakilkaya) [reply src/.php_cs:1]: This should be removed, I think semantic-release script couldn't run generate.sh file

File: src/git_push.sh (hunk 8 above)
Comment (Orkuncakilkaya) [reply src/git_push.sh:1]: We don't need this, semantic-release couldn't run generate.sh file

**Review by ilfa — APPROVED** [review]

**Pull request merged**